frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1404,13 +1431,15 @@ class Worksheet extends React.Component {
             textDeleted = false,
             fromDeleteCommand = false,
             uploadFiles = false,
+            searchRefresh = true,
```

Comment:
Can you add a comment here briefly explaining what this parameter means?

Reply:
added.